### PR TITLE
1194 Rename abstract methods. Use correct app.

### LINKED ISSUE #1194: Update ITs to not hit Stormpath administrators application
Update ITs.

### PR DESCRIPTION
**Pull request by josebarrueta**
Fixes #1194

### CODE REVIEW

**Comment by dogeared** [comment]
@josebarrueta - Failing CI due to Facebook IT failure in TCK suite. Your config is pointing to `dev.i` - Do you have a valid Facebook Directory configured and mapped to your `My Application`?

**Comment by josebarrueta** [comment]
Yeah forgot to configure it correctly.. done and re-running.

File: extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/AbstractSamlIT.groovy [hunk 1]
```diff
@@ -86,7 +86,7 @@ xu/vQr6stjuzJIsDNAtW1FlG8WALOMjV
         return client.getResource(samlPolicy.getIdentityProvider().href, SamlIdentityProvider)
     }
 
-    protected SamlIdentityProvider getSamlIdentityProviderForDefaultApplication() {
+    protected SamlIdentityProvider getSamlIdentityProviderForAdministratorsApplication() {
         def app = client.currentTenant.getApplications(Applications.where(Applications.name().eqIgnoreCase("Stormpath"))).asList().get(0)
```
Comment (mrioan) [inline extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/AbstractSamlIT.groovy]: you can do `client.currentTenant.getApplications(xxx).single()` (will throw if items != 1)

File: extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/RegisteredSamlServiceProviderIT.groovy [hunk 2]
```diff
@@ -377,7 +377,10 @@ class RegisteredSamlServiceProviderIT extends AbstractSamlIT {
 
     @Test
     void testSearchIdentityProviderCollectionOfRegisteredSamlServiceProvidersByDifferentProperties(){
-        def identityProvider = getSamlIdentityProviderForDefaultApplication()
+
+        def application = createTempApp()
+
```
Comment (mrioan) [inline extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/RegisteredSamlServiceProviderIT.groovy]: just being picky here; if possible remove this blank line

File: extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/RegisteredSamlServiceProviderIT.groovy [hunk 3]
```diff
@@ -425,7 +428,10 @@ class RegisteredSamlServiceProviderIT extends AbstractSamlIT {
 
     @Test
     void testDeletion() {
-        def identityProvider = getSamlIdentityProviderForDefaultApplication()
+
+        def application = createTempApp()
+
```
Comment (mrioan) [inline extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/RegisteredSamlServiceProviderIT.groovy]: just being picky here; if possible remove this blank line

File: extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/AbstractSamlIT.groovy (hunk 1 above)
Comment (josebarrueta) [inline extensions/httpclient/src/test/groovy/com/stormpath/sdk/impl/saml/AbstractSamlIT.groovy]: will do.

**Pull request merged**